core/modules/catalog-next/store/category/actions.ts

```diff
@@ -21,9 +21,11 @@ import chunk from 'lodash-es/chunk'
 import Product from 'core/modules/catalog/types/Product';
 import omit from 'lodash-es/omit'
 import config from 'config'
+import themeConfig from 'theme/theme.config'
 
 const actions: ActionTree<CategoryState, RootState> = {
-  async loadCategoryProducts ({ commit, getters, dispatch, rootState }, { route, category, pageSize = 50 } = {}) {
+  async loadCategoryProducts ({ commit, getters, dispatch, rootState }, { route, category } = {}) {
```

Comment:
this parameter needs to stay, and I think page parameter should be also added for pagination purposes